arangod/VocBase/VocbaseInfo.cpp

```diff
@@ -149,9 +148,10 @@ void CreateDatabaseInfo::toVelocyPack(VPackBuilder& builder, bool withUsers) con
   builder.add(StaticStrings::DatabaseName, VPackValue(_name));
   builder.add(StaticStrings::DataSourceSystem, VPackValue(_name == StaticStrings::SystemDatabase));
 
-  if (ServerState::instance()->isCoordinator()) {
+  if (ServerState::instance()->isCoordinator() ||
+      ServerState::instance()->isDBServer()) {
```

Comment:
Why is this necessary again?

Reply:
That is for  OneShard optimization. Without this DBServer can`t determine if vocbase is OneShard and apply optimization for analyzers loading